packages/core/src/cart/cart-action-creator.ts

```diff
@@ -0,0 +1,42 @@
+import { createAction, createErrorAction, ThunkAction } from '@bigcommerce/data-store';
+import { Observable, Observer } from 'rxjs';
+
+import { RequestOptions } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { InternalCheckoutSelectors } from '../checkout';
+import { cachableAction } from '../common/data-store';
+import ActionOptions from '../common/data-store/action-options';
+
+import { CartActionType, LoadCartAction } from './cart-actions';
+import CartRequestSender from './cart-request-sender';
+
+export default class CartActionCreator {
+    constructor(private _cartRequestSender: CartRequestSender) {}
+
+    @cachableAction
+    loadCart(
+        cartId: string,
+        options?: RequestOptions & ActionOptions,
+    ): ThunkAction<LoadCartAction, InternalCheckoutSelectors> {
+        return (store) => {
+            return new Observable((observer: Observer<LoadCartAction>) => {
+                const state = store.getState();
+                const host = state.config.getHost();
```

Comment:
`getHost` reads from config.meta. Do you know when we set the host?

Reply:
For catalyst this param is not required. For default host will be taken as **window.location.origin** when we are making an request. But I set it up directly

Reply:
If we don't need it maybe we can add it when it's needed, or else it just adds confusion.